plugins/services/src/js/components/forms/GeneralServiceFormSection.js

```diff
@@ -114,22 +74,32 @@ class GeneralServiceFormSection extends Component {
     this.setState({ convertToPodModalOpen: true });
   }
 
-  getAdvancedContainerSection() {
+  getAdvancedSettingsSection() {
     const { data = {}, errors, service } = this.props;
 
     if (service instanceof PodSpec) {
       return null;
     }
 
+    const initialIsExpanded = this.shouldShowAdvancedOptions();
```

Comment:
am I missing something, or did you create a new variable here which is only used once? If so why not call the method from JSX directly?